app/livechat/client/views/app/tabbar/chatRoomHistoryItem.html

```diff
@@ -0,0 +1,24 @@
+<template name='chatRoomHistoryItem'>
+	<li class='list-chat' id='{{_id}}' aria='{{v.token}}' >
+		<div class="history-div">
+			<div class="visitor-img">
+				<img src="/avatar/{{responseBy.username}}" alt="">
+			</div>
+			<div class="visitor-history-details">
+				<div class="visitor-details">
+					<h1>{{responseBy.username}}</h1>
+						<p>Close at time {{closedAt}}</p>
```

Comment:
Please, follow the prototype spec:
```suggestion
						<p>Close at {{closedAt}}</p>
```

Reply:
done